### ISSUE TITLE
Usage with ember-cli-concat

### ISSUE THREAD

**Issue by oskarrough** [ef4/ember-auto-import#87]
Hi, if you're using something like ember-cli-concat, which doesn't use the default vendor+app scripts, ember-auto-import seems to break because it can't find them in the DOM?

```
<script src="/assets/vendor.js"></script>
<script src="/assets/my-app.js"></script>
```

… but ember-cli-concat outputs:

```
<script async src="/assets/app.js"></script>
```

Is there any way to combine the two? Or does auto-import need the two files in the DOM?

**Comment by ef4** [comment]
Ah. We can fix this. The reason it's looking for the vendor script is that that's a heuristic for where to find any other chunks too. We can either come up with a more general heuristic, or make it configurable.

**Comment by ef4** [comment]
Closed by #89.
